### ISSUE TITLE
Unable to recognize a boost project

### ISSUE THREAD

**Issue by patricktiu** [OpenLiberty/liberty-dev-vscode-ext#3]
After importing this boost project, https://github.com/OpenLiberty/demo-devex into vscode, it is not rendering under the `LIBERTY DEV DASHBOARD`

The boost runtime dependency is defined within an `<profile/>` element in the project pom and not under project pom dependencies section.

```
               <profile>
                        <id>ol</id>
                        <activation>
                                <activeByDefault>true</activeByDefault>
                                <property>
                                        <name>boostRuntime</name>
                                        <value>ol</value>
                                </property>
                        </activation>
                        <dependencies>
                                <dependency>
                                        <groupId>boost.runtimes</groupId>
                                        <artifactId>openliberty</artifactId>
                                </dependency>
                        </dependencies>
                </profile>
```

**Comment by kathrynkodama** [comment]
Modifying the extension such that it detects boost via the `boost-maven-plugin`

**Comment by kathrynkodama** [comment]
fixed with #2
